QT/Naybrr/user.cpp

```diff
@@ -1,4 +1,5 @@
 #include "user.h"
```

Comment:
remove this

Reply:
The rest of the file yelled at me because it couldn't recognize User without the header.

Reply:
sorry the line numbers seem off, this seems fine, I might have been referring to any of the changes made for connecting to the db